package.json

```diff
@@ -75,6 +75,7 @@
     "test": "jest",
     "jest": "jest",
     "lint": "grunt lint",
+    "lint-changed": "git status --porcelain | grep -v '^??' | sed 's/^...//' | xargs eslint",
```

Comment:
Small preference for calling this `linc` which is shorter and also what I've had it called in previous projects.